lib/charts/cartesian_renderers/stackedline_chart_renderer.dart

```diff
@@ -0,0 +1,344 @@
+// Copyright 2017 Google Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+//
+
+part of charted.charts;
+
+class StackedLineChartRenderer extends CartesianRendererBase {
+  final Iterable<int> dimensionsUsingBand = const [];
+
+  final bool alwaysAnimate;
+  final bool showHoverCardOnTrackedDataPoints;
+  final bool trackDataPoints;
+  final bool trackOnDimensionAxis;
+  final int quantitativeScaleProximity;
+
+  bool _trackingPointsCreated = false;
+  List _xPositions = [];
+
+  // Currently hovered row/column
+  int _savedOverRow = 0;
+  int _savedOverColumn = 0;
+
+  int currentDataIndex = -1;
+
+  @override
+  final String name = "stacked-line-rdr";
+
+  StackedLineChartRenderer(
+      {this.alwaysAnimate: false,
+      this.showHoverCardOnTrackedDataPoints: false,
+      this.trackDataPoints: true,
+      this.trackOnDimensionAxis: false,
+      this.quantitativeScaleProximity: 5});
+
+  // Returns false if the number of dimension axes on the area is 0.
+  // Otherwise, the first dimension scale is used to render the chart.
+  @override
+  bool prepare(ChartArea area, ChartSeries series) {
+    _ensureAreaAndSeries(area, series);
+    if (trackDataPoints != false) {
+      _trackPointerInArea();
+    }
+    return area is CartesianArea;
+  }
+
+  @override
+  void draw(Element element, {Future schedulePostRender}) {
+    _ensureReadyToDraw(element);
+
+    var measureScale = area.measureScales(series).first,
+        dimensionScale = area.dimensionScales.first;
+
+    // We only support one dimension axes, so we always use the
+    // first dimension.
+    var x = area.data.rows
+        .map((row) => row.elementAt(area.config.dimensions.first))
+        .toList();
+
+    var acculumated = new List.filled(x.length, 0);
+
+    // Create lists of values used for drawing.
+    // First Half: previous values reversed (need for drawing)
+    // Second Half: current accumulated values (need for drawing)
+    var lines = series.measures.map((column) {
```

Comment:
Please reverse the order you draw the lines so that it matches the order we stack in the stacked bar chart so that the lower indexed measures are stacked on top (to match the order in legend)
    var lines = series.measures.toList().reversed.map((column) {
...